contents/handbook/growth/marketing/posthog-style-guide.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Style guide
+sidebar: Handbook
+showTitle: true
+---
+
+This style guide explains our guidelines for contributions to PostHog's documentation, tutorials, and blog.
+
+## General principles
```

Comment:
Two points I'd add in this section:
- Something around using direct, to-the-point language, avoiding passive tense?
- In addition to 'it's complicated', I really dislike 'it depends' when making a recommendation - we should make a stand!

I'd also consider changing the intros one to 'make your intros as short as possible' but that's just a personal preference.

Reply:
Agree on point 2, less convinced by point 1. Even the best written articles have some passive voice in them. I'd rather limit the guide to stuff that's (mostly) specific to us than general writing advice.